src/pages/blog/[id]/edit/editing/Editing.component.js

```diff
@@ -0,0 +1,79 @@
+import * as Styled from './editing.styled';
+import TiEditor from '@634750802/a-editor';
+import { Button, Checkbox, Input } from 'antd';
+import React from 'react';
+import { useEditContext, useEditMethods } from '../edit.context';
+
+const demoCategories = [
+  { label: 'cat1', value: 'cat1' },
```

Comment:
Now MockAPI of categories is available, so you can use it instead of the demo data

Reply:
That would be done in future PRs